pkg/controller/datavolume-controller.go

```diff
@@ -541,10 +533,30 @@ func (r *DatavolumeReconciler) reconcileClone(log logr.Logger,
 	pvcSpec *corev1.PersistentVolumeClaimSpec,
 	transferName string,
 	prePopulated bool,
-	pvcPopulated bool,
-	selectedCloneStrategy cloneStrategy) (reconcile.Result, error) {
+	pvcPopulated bool) (reconcile.Result, error) {
 
+	_, err := r.findSourcePvc(datavolume)
+	if err != nil {
+		if k8serrors.IsNotFound(err) {
```

Comment:
I wonder why there is a difference here vs the same lines in https://github.com/kubevirt/containerized-data-importer/pull/2405/commits/5686b4f1318a8bdce2f1f1b4899afa64082eb080

Reply:
in this version the !pvcPopulated was removed from the if in line 550. that code runs even if pvc populated so I needed to prevent it from running only if source pvc not found.

Reply:
ok